app/helpers/orchid/application_helper.rb

```diff
@@ -1,5 +1,33 @@
 module Orchid::ApplicationHelper
 
+  def active?(value, comparison=nil)
```

Comment:
It would be helpful for me is there is an example or two in a comment above this method!

I'm also trying to figure out how we might write some unit tests for this, because it seems like it would be really awesome to be able to test out if everything is working as we expect.  We would have to figure out how to stub in / mimic / trick `current_page?` to do it, I think?

Reply:
Okay, I can include a couple examples. I'll probably just copy a couple of the places where it's used in the Orchid code.

Not sure how we'd trick `current_page?`. I think for unit tests we'd almost have to stub in responses. Otherwise it looks like we'd need to use functional testing for regular `current_page?` behavior?

Reply:
This is super helpful, thanks!

Reply:
You're welcome! I hope the way I wrote it is clear enough.